src/main/java/frc/robot/subsystems/NoteVision.java

```diff
@@ -32,6 +36,7 @@
 public class NoteVision extends SubsystemBase {
     // Plot vision solutions
     public static final boolean PLOT_NOTES = true;
+    private static final double ALLOWED_NOTE_AMBIGUITY = .2;
```

Comment:
Unfortunately, "ambiguity" means something in the vision code, not related to distance.
How about "ALLOWED_POSITION_ERROR" or "MAX_POSITION_ERROR"?